src/core/state/currentSettings/connectedToHardhat.ts

```diff
@@ -13,10 +16,15 @@ export const connectedToHardhatStore = createStore<ConnectedToHardhatState>(
     setConnectedToHardhat: (connectedToHardhat) => {
       set({ connectedToHardhat });
     },
+
+    connectedToHardhatOp: false,
+    setConnectedToHardhatOp: (connectedToHardhatOp) => {
+      set({ connectedToHardhatOp });
+    },
   }),
   {
     persist: {
-      name: 'connectedToHardhat',
+      name: 'connectedToHardhatStore',
```

Comment:
why the change of name? 

i believe don't use the `Store` word at the end for store names

Reply:
probs a naming mistake then. will fix.